tower-test/src/macros.rs

```diff
@@ -0,0 +1,15 @@
+#[macro_export]
+macro_rules! assert_request_eq {
```

Comment:
might want docs here?

Reply:
Yeah. Probably... thanks for calling that out.